scripts/new-change

```diff
@@ -54,6 +54,10 @@ CHANGES_DIR = os.path.join(
 )
 TEMPLATE = """\
 # Type should be one of: feature, bugfix, enhancement, api-change
+# feature: A larger feature or change in behavior
```

Comment:
It would probably be good to add something  long the lines of:
``which typically entails a minor version bump.``